### PR TITLE
docs: adding Bearer Cloud integration doc

### PR DESCRIPTION
**Pull request by gmontard**
## Description
Adding a documentation section on Bearer cloud about the Jira and Slack integration

### CODE REVIEW

File: docs/guides/bearer-cloud.md [hunk 1]
```diff
@@ -93,9 +93,40 @@ Bearer Cloud automatically captures any scans run with a valid `api-key`. Subseq
...
+{% callout "warn" %}
+Findings on Bearer dashboard are marked resolved only when the associated code is fixed. For example, if the associated Jira ticket is closed, but no code fix has been applied, the finding will stay open on Bearer dashboard. The source of truth is always code.
+{% endcallout %}
+
+
+## Slack integration
+
+The Slack integration is available on the *Settings > Integrations* page.
+
+To use the integration, you must connect a Slack account and allow access to the required permissions through the OAuth login, then select a default channel where to receive notifications on new findings.
```
Comment (gotbadger) [inline docs/guides/bearer-cloud.md]:
```suggestion
To use the integration, you must connect a Slack account and allow access to the required permissions through the OAuth login, then select a default channel where you want to receive notifications on new findings.
```

File: docs/guides/bearer-cloud.md [hunk 2]
```diff
@@ -93,9 +93,40 @@ Bearer Cloud automatically captures any scans run with a valid `api-key`. Subseq
...
+
+Here is below an example of a Slack notification triggered by a new finding:
```
Comment (gotbadger) [inline docs/guides/bearer-cloud.md]:
```suggestion
Below an example of a Slack notification triggered by a new finding:
```

File: docs/guides/bearer-cloud.md [hunk 3]
```diff
@@ -93,9 +93,40 @@ Bearer Cloud automatically captures any scans run with a valid `api-key`. Subseq
...
+2. Link a finding to an existing Jira ticket.
+![Link Jira Ticket](/assets/img/jira-integration/link.png)
+
+Once a finding is associated with a Jira ticket, you can quickly see it in the interface, view the ticket status and go to the ticket. 
+
+![View Jira Ticket](/assets/img/jira-integration/view.png)
+
+
+{% callout "warn" %}
+Findings on Bearer dashboard are marked resolved only when the associated code is fixed. For example, if the associated Jira ticket is closed, but no code fix has been applied, the finding will stay open on Bearer dashboard. The source of truth is always code.
```
Comment (gotbadger) [inline docs/guides/bearer-cloud.md]:
```suggestion
Findings on Bearer Cloud are only marked resolved when the associated code is fixed. If the associated Jira ticket is closed, but no code fix has been applied, the finding will stay open. The source of truth is always the code.
```

**Comment by gmontard** [comment]
Thank you @gotbadger for the great review 🙏

**Review by gotbadger — APPROVED** [review]

**Pull request merged**